packages/ti_domaintools/data_stream/nod_feed/elasticsearch/ingest_pipeline/default.yml

```
@@ -1,13 +1,16 @@
 ---
 description: Pipeline for processing nod feed
 processors:
-- set:
-    field: domaintools.feed
-    value: 'nod'
 - json:
+    if: ctx?.message != null
```

```suggestion
    if: ctx.message != null
```
(throughout; `ctx` is always non-null)